scripts/prove-scoped-package-access/main_test.go

```
@@ -0,0 +1,286 @@
+					if r.Header.Get("Authorization") != "Bearer scoped-secret" {
+						t.Error("repository scope queried with the wrong credential")
+					}
+					if tc.fault == "invalid-token" {
+						w.WriteHeader(401)
+						return
+					}
+					count := 1
+					if tc.fault == "wide-token" || tc.fault == "partial-token" {
+						count = 2
+					}
+					fmt.Fprintf(w, `{"total_count":%d,"repositories":[{"full_name":"devantler-tech/wedding-app"}]}`, count)
```

golangci-lint (2.13.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)